src/domain/event/eventCard/EventCard.tsx

```diff
@@ -26,55 +29,42 @@ const EventCard: React.FC<Props> = ({ event, onClick }) => {
   const image = event.images[0]?.url;
   const time = getEventStartTimeStr(event, locale, t);
 
-  const handleClick = () => {
-    if (onClick) {
-      onClick(event.id || '');
-    }
-  };
-
-  const handleKeyDown = (event: React.KeyboardEvent) => {
-    if (event.key === 'Enter') {
-      handleClick();
-    }
-  };
-
   return (
-    <div
-      role="button"
-      className={styles.eventCard}
-      tabIndex={0}
-      onClick={handleClick}
-      onKeyDown={handleKeyDown}
-      aria-label={t('event:eventCard.ariaLabelOpenEvent', {
-        eventName: name,
-      })}
-    >
-      <div
-        className={styles.imageWrapper}
-        style={{
-          backgroundImage: `url(${
-            image || getEventPlaceholderImage(id || '')
-          })`,
-        }}
-      ></div>
-      <div className={styles.contentWrapper}>
-        <div className={styles.titleWrapper}>
-          <div className={styles.title}>{name}</div>
-          <div className={styles.description}>{description}</div>
-        </div>
-        <div className={styles.occurrenceInfoWrapper}>
-          <div className={styles.textWithIcon}>
-            <IconClock />
-            {time}
+    <Link href={link}>
```

Comment:
I think that the dynamic routing is not working properly here. When I'm clicking event card i get this error message
<img width="880" alt="Näyttökuva 2020-10-9 kello 14 34 50" src="https://user-images.githubusercontent.com/24706814/95578529-b0797f00-0a3c-11eb-8956-451f0febfc7a.png">
And on event page
<img width="1435" alt="Näyttökuva 2020-10-9 kello 14 30 34" src="https://user-images.githubusercontent.com/24706814/95578564-bff8c800-0a3c-11eb-956e-b97814205cb9.png">

You can fix this by adding as prop to Link
e.g. 
```
<Link
      href={`${ROUTES.EVENT_DETAILS.replace(':id', '[eventId]')}?eventId=${
        event.id
      }`}
      as={ROUTES.EVENT_DETAILS.replace(':id', event.id)}
    >
```
More info can be found here https://github.com/isaachinman/next-i18next/issues/413

Reply:
hmm next docs mention that this props is used before version Next.js 9.5.3. Need to check if updating next would help 🤔

Reply:
And I think I have seen that error evenr before when running locally

Reply:
The problem existed already before this PR. next-i18next still seems to use 'as' prop internally. Here's source code for that component https://github.com/isaachinman/next-i18next/blob/master/src/components/Link.tsx 

But hopefully just updating next.js to latest version does the magic